coolify/setup.sh

```
@@ -52,15 +51,60 @@ else
+
+# Create redis logs subdirectory with correct ownership
+if [ ! -d "logs/redis" ]; then
+    mkdir -p logs/redis
+    print_success "Created logs/redis/ directory"
+else
+    print_warning "logs/redis/ directory already exists"
+fi
+
+# Try to set correct ownership (UID 999 = redis user in container)
+if command -v sudo &> /dev/null && [ "$OSTYPE" != "msys" ] && [ "$OSTYPE" != "win32" ]; then
+    sudo chown -R 999:999 logs/redis 2>/dev/null && \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)